proof/proof/inc/TProof.h

```diff
@@ -1071,7 +1071,7 @@ friend class TXProofServ;     // to access EUrgent
    static Int_t         GetParameter(TCollection *c, const char *par, Double_t &value);
 
    ClassDefOverride(TProof,0)  //PROOF control class
-};
+} R__DEPRECATED(6,38, "Legacy interactive distributed processing engine. Use RDataFrame and DistRDF instead.");
```

Comment:
```suggestion
} R__DEPRECATED(6,38, "Legacy interactive distributed processing engine. Use RDataFrame instead.");
```

`DistRDF` is an internal/colloquial name. I would not use it in a public-facing warning. Alternatively, we could write `RDataFrame (local or distributed) instead`.

Reply:
makes sense. Thanks for the comment.